File: Code Chef/JavaScript/300-399/339 Max minus Min MAXDIFFMIN.js
process.stdin.setEncoding("utf-8");

const strToInt = (strArray) => {
    strArray.shift();
    const intArray = strArray.map((strItem) => strItem.split(" ").map(Number));
    
    return intArray;
};

const minMaxDifference = (min, max) => {
    const difference = max - min;
    console.log(difference);
};

process.stdin.on("data", (input) => {
    const testcases = strToInt(input.trim().split("\n"));
    
    testcases.forEach((testcase) => {
        const [num1, num2, num3] = testcase;
        const min = Math.min(num1, num2, num3);
        const max = Math.max(num1, num2, num3);
        
        minMaxDifference(min, max);
    });
});

File: Code Chef/C Programming/300-399/339 Max minus Min MAXDIFFMIN.c
#include <stdio.h>
int main(void)
{
	int t,a,b,c,max,min;
	scanf("%d",&t);
	for(int i=0;i<t;i++)
	{
	    scanf("%d%d%d",&a,&b,&c);
	    printf("%d\n",(c-a));
	}
	return 0;
}